atomic-write.cabal

```diff
@@ -72,10 +72,17 @@ library
 
 test-suite atomic-write-test
   type: exitcode-stdio-1.0
-  hs-source-dirs: spec, src
+  hs-source-dirs: spec
   main-is: Spec.hs
 
+  other-modules:       System.AtomicWrite.Writer.ByteStringSpec
+                     , System.AtomicWrite.Writer.ByteStringBuilderSpec
+                     , System.AtomicWrite.Writer.LazyByteStringSpec
+                     , System.AtomicWrite.Writer.StringSpec
+                     , System.AtomicWrite.Writer.TextSpec
+
   build-depends:       base >= 4.5 && < 5.0
+                     , atomic-write
                      , temporary
```

Comment:
@acamino about:

> I double check them. Those modules are defined as specs. I'm not requiring the modules defined at src. :wink:
If I don't add those dependencies (other-modules) we'll end with a tarball like this https://hackage.haskell.org/package/atomic-write-0.2.0.5/src/ where the spec modules are excluded. What do you think?

I was taking about the dependencies (e.g. time, temporary)  not the exported modules :)